cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                self.stdout.write("Database unavailable")
```

Comment:
It would be better to add a delay before retrying the connection to the database. This is to avoid unnecessary CPU usage while waiting for the database to become available.